regression/esbmc-cpp/cpp/ dangling_ptr/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+class Foo {
+  public:
+    Foo() {value = 0;};
+    void Execute();
+    void Increment();
+  private:
+    int value;
+};
+
+void Foo::Execute() {
+        printf("Executing...");
+}
+
+void Foo::Increment() {
+    value++;
+}
+
+int main()
+{
+  Foo *foo = new Foo();
+
+  foo->Increment();  // Incrementing the value
+
+  foo->Execute(); // Executing
+
```

Comment:
Can you also create another test case that adds the delete to mark it as VERIFICATION SUCCESSFUL?